### PR TITLE
`to_list()` accepts NoneType

### LINKED ISSUE #2: `.to_list(None)`
https://github.com/L0RD-ZER0/Motor-Types/blob/379b709157f524ff2b448ca4c28f34eb127a6441/motor-stubs/core.pyi#L734

https://motor.readthedocs.io/en/stable/api-tornado/cursors.html#motor.motor_tornado.MotorCommandCursor.to_list

### PR DESCRIPTION
**Pull request by haoyun**
resolves #2.

### CODE REVIEW

File: motor-stubs/core.pyi [hunk 1]
```diff
@@ -731,7 +731,7 @@ class AgnosticBaseCursor(AgnosticBase):
     def get_io_loop(self) -> _IO_Loop: ...
     async def next(self) -> _Document: ...
     def next_object(self) -> _Document: ...
-    async def to_list(self, length: int) -> typing.List[_Document]: ...
+    async def to_list(self, length: typing.Optional[int]) -> typing.List[_Document]: ...
```
Comment (L0RD-ZER0) [inline motor-stubs/core.pyi:734]: According to my observations, we do need to provide a value to that parameter or python will error out. However, it does accept none as a value.

Comment (L0RD-ZER0) [reply motor-stubs/core.pyi:734]: A better alternative would be to use `typing.Union[int, None]`

**Review by L0RD-ZER0 — APPROVED** [review]
After some thought, I've decided to merge it as `Optional` in python typing doesn't say that it's an optional argument. If need be, we can always modify it later.

**Comment by haoyun** [comment]
> `Optional` in python typing doesn't say that it's an optional argument.

Yes, In python, optional[t] is the same as union[t, none]. In fastapi, explicit union[t, none] or t|None is recommended.

**Pull request merged**